File: micro-services-springboot-employee-service/src/main/java/com/example/employeeservice/service/EmployeeService.java
package com.example.employeeservice.service;

import com.example.employeeservice.dto.EmployeeDto;
import com.example.employeeservice.entity.Employee;

public interface EmployeeService {

	EmployeeDto saveEmployee(EmployeeDto employeeDto);
	EmployeeDto getEmployee(long employeeId);
}

File: react-front-end/src/service/EmployeeService.js
import axios from 'axios';

const EMPLOYEE_SERVICE_BASE_URL = 'http://localhost:9191/employee-service/api/employee';

const EMPLOYEE_ID = 4;

class EmployeeService{
    getEmployee(){
        return axios.get(EMPLOYEE_SERVICE_BASE_URL + '/' + EMPLOYEE_ID);
    }
}


export default new EmployeeService